sysml.library/Domain Libraries/Geometry/SpatialItems.sysml

```diff
@@ -81,18 +59,15 @@ package SpatialItems {
 		 * The coordinateFrames of the compositeItems are nested in the coordinateFrame of the CompoundSpatialItem, 
 		 * and, by default, they have the same localClock as the CompoundSpatialItem.
 		 */
-		item <spatialSubitems> componentItems : SpatialItem[0..*] ordered :> subitems {
+		item componentItems : SpatialItem[0..*] ordered :> subitems {
 			item nestedLocalClock :>> localClock default localClock;
 			attribute nestedBasisDirections : VectorQuantityValue[3];
 			attribute nestedOrigin : VectorQuantityValue[0..1];
 			attribute nestedCoordinateFrame :>> coordinateFrame {
 				attribute :>> transformation[1] {
 				 	attribute :>> source = coordinateFrame;
 				 	attribute :>> target = nestedCoordinateFrame;
-				 	//attribute :>> basisDirections = nestedBasisDirections;
-				 	//attribute :>> origin = nestedOrigin;
```

Comment:
If `basisDirections` and `origin` are removed from `nestedCoordinateFrame`, then `nestedBasisDirections` and `nestedOrigin` need to be removed from `componentItems`. The latter were only there to make it easier to set the former.